patches/kaggle_secrets.py

```diff
@@ -57,11 +58,21 @@ def _make_post_request(self, data):
             raise BackendError('Unexpected response from the service.') from e
 
     def get_bigquery_access_token(self):
+        """
```

Comment:
Nit:  Summary line on this first line, then blank line, then more.  E.g.:

```
"""Retrieves BigQuery access token information from the UserSecrets service.

This returns the token for the current kernel as well as its expiry (abs time) if it
is present.

Example:
   ...
```

Also provide type information (is the expiry a number of seconds since epoch?  a datetime? ...) and explain what the returned expiry is when it is not present (`None` I suppose).

Reply:
Added typing info.